templates/certified/desktops.html

```diff
@@ -1,195 +1,43 @@
-{% extends "certified/base_certification.html" %}
+{% extends "templates/base.html" %}
 
 {% block title %}Certified desktops{% endblock %}
+{% block meta_description %}Ubuntu certifies thousands of desktops to guarantee they work well with Ubuntu.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1OWoADcvZ-KQX-BzsCS--pvs9_pY3E-Lb8Ba6a-T-AZ8/edit{% endblock %}
 
-{% block meta_description %}Desktops that have been certified for use with Ubuntu.{% endblock meta_description %}
+{% block outer_content %}
 
-{% block content %}
-
-<form class="js-search-results">
-  <section class="p-strip--suru-topped">
-    <div class="u-fixed-width">
-      <h1 class="u-sv3">Ubuntu certified desktops</h1>
-    </div>
-    <div class="u-fixed-width">
-      <div class="p-search-box">
-        <input class="p-search-box__input" type="text" name="q" value="{{ query or '' }}" placeholder="Search hardware">
-        <button type="submit" class="p-search-box__button"><i class="p-icon--search"></i></button>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-3">
-        <h2 class="p-heading--4">Filter</h2>
-      </div>
-      <div class="col-4">
-        <h2 class="p-heading--4">
-          {% if total_results %}
-          {% if total_results and total_results > 1 %}
-          {{ offset + 1 }}
-          &ndash;
-          {% if offset + limit > total_results %}
-          {{ total_results }}
-          {% else %}
-          {{ offset + limit }}
-          {% endif %}
-          of
-          {% endif %}
-          {{ total_results }} result{% if total_results != 1 %}s{% endif %}
-          {% endif %}
-        </h2>
-      </div>
-      <div class="col-5 u-align--right">
-        {% if total_results > 0 %}
-        <p style="display: inline-block; padding-right: 0.5rem;">Results per page</p>
-        <select id="page-size-top" class="p-results-per-page" name="limit" aria-label="Results per page" style="max-width: 5.3rem; min-width: 5.3rem;" data-limit="{{ limit }}">
-          <option value="20"{% if limit == 20 %}selected{% endif %}>20</option>
-          {% if total_results > 20 %}
-          <option value="40"{% if limit == 40 %}selected{% endif %}>40</option>
-          {% endif %}
-          {% if total_results > 40 %}
-          <option value="60"{% if limit == 60 %}selected{% endif %}>60</option>
-          {% endif %}
-          <option value="{{ total_results }}" {% if limit == total_results %}selected{% endif %}>All</option>
-        </select>
-        {% endif %}
-      </div>
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu Certified desktops</h1>
```

Comment:
I have some questions regarding this. There is a bunch of places where we have lowerase certified (/desktop/developers, /desktop/partners, /appliance/community), which kind of make sense to me, because it's the verb certify rather than a proper noun, and other places where it is capitalised (previous /certified pages, legal/ubuntu-advantage-service-description). Do we want to make these consistent by choosing one side or the other? Or are these maybe 2 different things?

Reply:
Good spot with the other pages - I think we should make it consistently lower case as you're right it makes sense that way. Maybe we can make another issue to change them or if it's not many places could do it here - what do you think?